src/MfaVerifyController.js

```diff
@@ -89,12 +89,12 @@ function (Okta, Checkbox, BaseLoginController, CookieUtil, TOTPForm, YubikeyForm
         }
 
         // Remember Device checkbox resides outside of the Push and TOTP forms.
-        if (this.settings.get('features.rememberDevice')) {
+        if (this.settings.get('features.rememberDevice') && this.options.appState.get('allowRememberDevice')) {
```

Comment:
We will not need the rememberDevice config anymore, right? (Since all the logic resides in the API)